src/webview/MessageList.js

```diff
@@ -334,15 +334,15 @@ export default connect<SelectorProps, _, _>((state, props: OuterProps) => {
   // it'd be better to set an example of the right general pattern.
   const backgroundData: BackgroundData = {
     alertWords: state.alertWords,
+    allImageEmojiById: getAllImageEmojiById(state),
     auth: getAuth(state),
     debug: getDebug(state),
     flags: getFlags(state),
     mute: getMute(state),
     ownEmail: getOwnEmail(state),
     ownUserId: getOwnUser(state).user_id,
-    allImageEmojiById: getAllImageEmojiById(state),
-    subscriptions: getSubscriptions(state),
     twentyFourHourTime: getRealm(state).twentyFourHourTime,
+    subscriptions: getSubscriptions(state),
```

Comment:
`"t" < "s"`?

Reply:
Ha, thanks for the catch. The two lists disagreed, but then I settled it the wrong way :)